Assets/VRTK/Prefabs/SnapDropZone/VRTK_SnapDropZone.cs

```diff
@@ -111,7 +111,7 @@ public enum SnapTypes
         protected GameObject highlightEditorObject = null;
 
         protected List<VRTK_InteractableObject> currentValidSnapInteractableObjects = new List<VRTK_InteractableObject>();
-        protected VRTK_InteractableObject currentSnappedObject = null;
+        public VRTK_InteractableObject currentSnappedObject = null;
```

Comment:
Why has this been made public?

if there is a valid reason for it to be public (and serialised so it shows up in the unity inspector) then it needs moving to the public block and also needs a tooltip.